### PR TITLE
chore: bump category api versions

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

Absorb latest tag release from category api.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — APPROVED** [review]

File: packages/amplify-util-mock/package.json [hunk 1]
```diff
@@ -50,19 +50,19 @@
...
-    "@aws-amplify/graphql-relational-transformer": "^1.1.0-cdkv2.2",
-    "@aws-amplify/graphql-searchable-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-core": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-interfaces": "^2.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-migrator": "^2.1.0-cdkv2.2",
+    "@aws-amplify/graphql-auth-transformer": "^2.1.0-cdkv2.3",
+    "@aws-amplify/graphql-default-value-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-function-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-http-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-index-transformer": "^1.1.0-cdkv2.3",
```
Comment (awsluja) [inline packages/amplify-util-mock/package.json:57]: is this correct?

File: packages/amplify-util-mock/package.json [hunk 2]
```diff
@@ -50,19 +50,19 @@
...
+    "@aws-amplify/graphql-maps-to-transformer": "^2.1.0-cdkv2.3",
+    "@aws-amplify/graphql-model-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-predictions-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-relational-transformer": "^1.1.0-cdkv2.3",
```
Comment (awsluja) [inline packages/amplify-util-mock/package.json:61]: here too

File: packages/amplify-util-mock/package.json [hunk 3]
```diff
@@ -50,19 +50,19 @@
...
+    "@aws-amplify/graphql-searchable-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-transformer-core": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-transformer-interfaces": "^2.1.0-cdkv2.2",
+    "@aws-amplify/graphql-transformer-migrator": "^2.1.0-cdkv2.3",
```
Comment (awsluja) [inline packages/amplify-util-mock/package.json:65]: and here

**Review by awsluja — APPROVED** [review]

File: packages/amplify-util-mock/package.json (hunk 1 above)
Comment (sobolk) [reply packages/amplify-util-mock/package.json:57]: yes. we had 4 tag releases in data repo and the list of packages that got released was growing over time. hence last component is not in line.

**Pull request merged**